### ISSUE TITLE
PHP Couchbase 2.1.0 is broken PECL module

### ISSUE THREAD

**Issue by madhancock** [Homebrew/homebrew-php#3167]
## Description

The version of the Couchbase module added in #3091 is from PECL. This version causes fatal errors during view queries. Documented here: https://forums.couchbase.com/t/cannot-install-sdk-v2-1-0-on-mac-osx/6096/6

It doesn't appear that a patched release ever made it onto PECL so the solution at present would be to revert to 2.0.7.

## Context and Logs

```
Fatal error: Cannot use object of type stdClass as array in [CouchbaseNative]/CouchbaseBucket.class.php on line 284
```

Parameter | Value
--------- | ------------------
**OS X Version:**       | 10.11.4
**Homebrew Version:**   | 0.9.9
**PHP Version in use:** | 5.6

**Comment by avsej** [comment]
@madhancock this should be fixed by #3173

**Comment by javian** [comment]
The fix has now been merged @madhancock and I'll close this issue. Please give it a test run and we can reopen it if it doesn't work. Thanks for the patch @avsej !
